### DISCUSSION TITLE
Change speed while playing

### DISCUSSION THREAD

**Discussion by yarona1993** [asciinema/discussions#163] [Q&A]
Is it possible to change the speed while player is running without restart the whole cast?

**Comment by ku1ik** [comment]
Support for this was present in player v2 but hasn't been ported to v3 (which was rewrite from scratch).

**Comment by dateolive** [comment]
> Support for this was present in player v2 but hasn't been ported to v3 (which was rewrite from scratch).

Like speed-up (>) / slow-down (<) keys?
Wouldn't it be better if the video had a speed option?

**Comment by ku1ik** [comment]
What do you mean by "video having a speed option"?

**Comment by dateolive** [comment]
> What do you mean by "video having a speed option"?

When the video is playing, I wish there was an option to make the video play faster.
Like 'x 1','x 1.5','x 2'.
